### PR TITLE
Align table checkbox

### PR DESCRIPTION
**Pull request by yesmeck**
#2651 #4962 

设计师给的标注：

### CODE REVIEW

**Comment by afc163** [comment]
- [ ] When cursor hover at arrow, it should be hand.

**Comment by afc163** [comment]
image

I think it is too wider.

**Comment by afc163** [comment]
image

It should not have a selected background in "反选"。

**Comment by yesmeck** [comment]
这里能不能 at 到设计师的？

**Comment by afc163** [comment]
And 『全选』and『反选』should be in Table.locale.

**Comment by infeng** [comment]
"反选" have a selected background because of [this commit](https://github.com/ant-design/ant-design/commit/6fdb39038b2b4620bd7e3f1d8e54e349f9e8283f)

File: components/table/style/index.less [hunk 1]
```diff
@@ -138,12 +138,17 @@
...
+    padding-right: 0;
+  }
+
   &-thead > tr > th.@{table-prefix-cls}-selection-column,
   &-tbody > tr > td.@{table-prefix-cls}-selection-column,
   &-expand-icon-th,
   &-row-expand-icon-cell {
     text-align: center;
-    width: 48px;
+    width: 60px;
```
Comment (afc163) [inline components/table/style/index.less]: 62px ?

File: components/table/style/index.less [hunk 2]
```diff
@@ -435,7 +440,9 @@
   }
 
   &-selection {
-    width: 40px;
+    &-selection-all {
```
Comment (afc163) [inline components/table/style/index.less]: &-select-all

File: components/table/style/index.less [hunk 3]
```diff
@@ -459,8 +466,10 @@
     }
 
     &-down {
+      cursor: pointer;
       padding: 0;
       display: inline-block;
+      vertical-align: middle;
```
Comment (afc163) [inline components/table/style/index.less]: 这一行让表头高度变高了。

**Comment by yesmeck** [comment]
Updated.

File: components/table/style/index.less [hunk 4]
```diff
@@ -459,6 +466,7 @@
     }
 
     &-down {
+      cursor: pointer;
       padding: 0;
       display: inline-block;
```
Comment (afc163) [inline components/table/style/index.less]: 呃，可以这样修复垂直对齐：

```css
vertical-align: middle;
line-height: 1;
```

**Review by afc163 — APPROVED** [review]

**Pull request merged**